frontend/src/theme/globalStyles.ts

```diff
@@ -658,21 +658,24 @@ export const globalStyles = (theme: Theme): any => css`
     display: none !important;
   }
 
-  // Make scrollbars awesome in Chrome
+  // Make scrollbars awesome in Chrome.
+  // The scrollbar settings look and behave strangely with glide-data-grid.
+  // Therefore, we exclude dvn-scroller (part of interactive tables) here.
+  // It is not possible to reset or fix the scrollbar settings directly at the component.
 
-  ::-webkit-scrollbar {
+  :not(.dvn-scroller)::-webkit-scrollbar {
```

Comment:
Just checking, is `dvn-scroller` a class specific to `glide-data-grid`?

Reply:
yep, `dvn-scroller` is from `glide-data-grid`. It is the component that holds the scrollbars to scroll on the canvas area.